TYPO3.Flow/Classes/TYPO3/Flow/Configuration/ConfigurationManager.php

```diff
@@ -651,24 +651,51 @@ protected function postProcessConfiguration(array &$configurations)
         foreach ($configurations as $key => $configuration) {
             if (is_array($configuration)) {
                 $this->postProcessConfiguration($configurations[$key]);
```

Comment:
Probably would be, would also enable replacing keys (could be done in iterating as well, but not so nice). But IMHO that's a topic for another PR. This change won't impact performance much compared to what we have now, so....